cmd/api/src/database/migration/migrations/v6.0.0.sql

```diff
@@ -1,83 +0,0 @@
-        '{
-          "enabled": false
-        }', current_timestamp, current_timestamp)
-ON CONFLICT DO NOTHING;
-
--- Add Prune TTLs
-INSERT INTO parameters (key, name, description, value, created_at, updated_at)
-VALUES ('prune.ttl', 'Prune Retention TTL Configuration Parameters',
-        'This configuration parameter sets the retention TTLs during analysis pruning.', '{
-    "base_ttl": "P7D",
-    "has_session_edge_ttl": "P3D"
-  }', current_timestamp, current_timestamp)
-ON CONFLICT DO NOTHING;
-
--- Add Reconciliation to parameters and remove from feature_flags
-INSERT INTO parameters (key, name, description, value, created_at, updated_at)
-VALUES ('analysis.reconciliation', 'Reconciliation',
-        'This configuration parameter enables / disables reconciliation during analysis.', format('{"enabled": %s}',
-                                                                                                  (SELECT COALESCE(
-                                                                                                            (SELECT enabled FROM feature_flags WHERE key = 'reconciliation'),
-                                                                                                            TRUE))::text)::json,
-        current_timestamp, current_timestamp)
-ON CONFLICT DO NOTHING;
--- must occur after insert to ensure reconciliation flag is set to whatever current value is
-DELETE
-FROM feature_flags
-WHERE key = 'reconciliation';
-
--- Grant the Read-Only user SavedQueriesRead permissions
-INSERT INTO roles_permissions (role_id, permission_id)
-VALUES ((SELECT id FROM roles WHERE roles.name = 'Read-Only'),
-        (SELECT id FROM permissions WHERE permissions.authority = 'saved_queries' and permissions.name = 'Read'))
-ON CONFLICT DO NOTHING;
-
--- Add OIDC Support feature flag
-INSERT INTO feature_flags (created_at, updated_at, key, name, description, enabled, user_updatable)
-VALUES (current_timestamp,
-        current_timestamp,
-        'oidc_support',
-        'OIDC Support',
-        'Enables OpenID Connect authentication support for SSO Authentication.',
-        false,
-        false)
-ON CONFLICT DO NOTHING;
-
--- Update existing Edge tables with an additional constraint to support ON CONFLICT upserts
-do
-$$
-  begin
-    -- Update existing Edge tables with an additional constraint to support ON CONFLICT upserts
-    alter table edge
-      drop constraint if exists edge_graph_id_start_id_end_id_kind_id_key;
-    alter table edge
-      add constraint edge_graph_id_start_id_end_id_kind_id_key unique (graph_id, start_id, end_id, kind_id);
-  exception
-    -- This guards against the possibility that the edge table doesn't exist, in which case there's no constraint to
-    -- migrate
-    when undefined_table then null;
-  end
-$$;
```

Comment:
This does not appear to be in our schema.sql, are we sure it's handled in our graph schema?

Reply:
Shows up in my pgdumps, we're good to go